FlavorTrack/Services/BusinessDataService.swift

```diff
@@ -29,51 +32,55 @@ final class BusinessDataService {
 	/// 	- nearby: A string representing the nearby address.
 	/// 	- businessType: A string representing the business type.
 	/// 	- onComplete: A completion handler that takes in a `Result<T, BusinessDataService.NetworkError>` object.
-	func fetchData<T: Decodable>(nearby address: String, businessType: String,
-								 onComplete: @escaping (Result<T, BusinessDataService.NetworkError>) -> Void) -> Void {
+	func fetchData<T: Decodable>(
+		nearby address: String,
+		businessType: String,
+		onComplete: @escaping (Result<T, BusinessDataService.NetworkError>) -> Void) {
+
+		// swiftlint:disable line_length
```

Comment:
Again, adding exception is not ideal. It's a good practice to not have any exception throughout the code.